jetty-documentation/src/main/asciidoc/administration/logging/jetty-server-dump.adoc

```diff
@@ -80,26 +80,26 @@ The Server MBean has a `dump()` method, which dumps everything, plus a `dumpStdE
 This is a dump of the OneServletContext embedded example with extra threadpool information:
 
 ....
-Server@59906517{STARTED}[9.4.32-SNAPSHOT] - STARTED
-+= QueuedThreadPool[qtp1740189450]@67b92f0a{STARTED,8<=8<=200,i=5,r=4,q=0}[ReservedThreadExecutor@77e4c80f{s=0/4,p=0}] - STARTED
-|  += ReservedThreadExecutor@77e4c80f{s=0/4,p=0} - STARTED
+Server@5ed828d{STARTED}[9.4.32-SNAPSHOT] - STARTED
++= QueuedThreadPool[qtp731260860]@2b9627bc{STARTED,8<=8<=10,i=5,r=1,q=0}[ReservedThreadExecutor@77e4c80f{s=0/1,p=0}] - STARTED
+|  += ReservedThreadExecutor@77e4c80f{s=0/1,p=0} - STARTED
 |  +> threads size=8
-|  |  +> 12 qtp1740189450-12 SELECTING  RUNNABLE 5
-|  |  +> 14 qtp1740189450-14-acceptor-0@4c78251c-ServerConnector@76707e36{HTTP/1.1, (http/1.1)}{0.0.0.0:8080} ACCEPTING  RUNNABLE 3
-|  |  +> 16 qtp1740189450-16 IDLE  TIMED_WAITING 5
-|  |  +> 18 qtp1740189450-18 IDLE  TIMED_WAITING 5
-|  |  +> 15 qtp1740189450-15 SELECTING  RUNNABLE 5
-|  |  +> 19 qtp1740189450-19 IDLE  TIMED_WAITING 5
-|  |  +> 17 qtp1740189450-17 IDLE  TIMED_WAITING 5
-|  |  +> 13 qtp1740189450-13 IDLE  TIMED_WAITING 5
+|  |  +> 13 qtp731260860-13 RUNNABLE 5 SELECTING
```

Comment:
I would not mind to have a different logging of this, namely: name, state, tid, prio, like this:
`qtp731260860-13 RUNNABLE SELECTING tid=13 prio=5`
so that the important information (name and state) are at the beginning.

Reply:
This change would make the dump would not have a consistent ordering between threads in known methods and those not.

The `SELECTING` is really just a reduction of the stacktrace, we have a stackframe in the method `SelectorProducer.select()` so we abbreviate this to `SELECTING`. So it makes sense to me that we would want to have this in the same position as we would have the stacktrace (the last position).

So I think this would be a better format
`qtp731260860-13 RUNNABLE tid=13 prio=5 SELECTING`

this would be a format of
`name` `state` `id` `priority` `{KNOWN / stackTraceFirstLine / fullStackTrace}`

Reply:
@lachlan-roberts agreed with your latest proposal.